src/embedded_flash/store/mod.rs

```diff
@@ -728,6 +770,22 @@ impl<C: StoreConfig> Store<BufferStorage, C> {
     pub fn set_erase_count(&mut self, page: usize, erase_count: usize) {
         self.initialize_page(page, erase_count);
     }
+
+    /// Checks whether all deleted sensitive entries have been wiped.
+    pub fn check_wipe(&self) {
```

Comment:
`#cfg(test)`? Nit: I'd also rename it `check_wiped`.

Reply:
This `impl` is under `cfg(feature = "std")` and documented as not for production. This is used both in unit tests and in fuzzing. I don't think you can build a library such that `cfg(test)` is enabled.

Renamed to `check_wiped`.